Sources/Presentation/Presenter/AlertControllerFactory.swift

```diff
@@ -94,6 +89,26 @@ public class AlertControllerFactory {
         return alertController
     }
     
+    /**
+     Converts the entity class into the `UIAlertController`.
+     - TODO: Rename.
+     
+     - Parameters:
+       - updateDetail: The entity class.
+     
+     - Returns: `UIAlertController` with the default actions and the completion handler.
+     */
+    public class func from(updateDetail: UpdateDetail) -> UIAlertController {
+        let factory = AlertControllerFactory(texts: DialogTexts(
+            title: updateDetail.title,
+            message: updateDetail.message
+        ))
+        // TODO: 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Scior/Uppsala/blob/3e3bf287e532edf7eff7664ca9a2a4565256aa13/Sources/Presentation/Presenter/AlertControllerFactory.swift#L106"></span>TODOs should be resolved.</td>
    </tr>
  </tbody>
</table>